build/FLExBridge.proj

```diff
@@ -27,8 +27,10 @@
 	<Import Project="$(MSBuildTasksTargets)" Condition="Exists('$(MSBuildTasksTargets)')"/>
 	<Import Project="NuGet.targets"/>
 	<Import Project="WixPatchableInstaller.targets" Condition="'$(OS)'=='Windows_NT'"/>
-	<Import Project="$(GitVersionTaskAltPath)/GitVersionTask.alt.targets"
-		Condition="Exists('$(GitVersionTaskAltPath)/GitVersionTask.alt.targets') And !$(IgnoreGitVersionTask)"/>
+	<Import Project="$(GitVersionTaskPath)/GitVersion.MsBuild.targets"
+		Condition="Exists('$(GitVersionTaskPath)/GitVersion.MsBuild.targets') And !$(IgnoreGitVersionTask)"/>
+	<Import Project="$(GitVersionTaskPath)/GitVersion.MsBuild.props"
```

Comment:
Might be better to reverse importing .props and .targets - usually .props gets imported first. Probably doesn't matter though...